testscommon/testdata/outportBlockHeaderV1.json

```diff
@@ -0,0 +1,83 @@
+{
+  "blockData": {
+    "headerBytes": "CInyBBIgoMNT6N8hiIvxegFdiANPtl4EfB8LcV5KOdBzbTzKM4kaMCPxCBfqk7yKetaKgXGMgaqBCsL+xOWODbZlwMKTSVxg8Rr7lua0dhL59ThHP7PHFiIwYKEyEvmF3qETcM+ZPj6BA7VYf7UOxcK14nu1JKzhZwQEwrZI5mZImiXL5Vy/ZLIFKgj/////////fzjq0aj5BUCX8gRIBVowLQKZCT5IECw4ga6mpRu1rNCZhssXQk03wbH+SOhOIGyMOa04tOcJ0dJB+GRgTIUTYjBVJ1f6mKAxo2HQugt414hrMFG66tklXVdlzW5MCJ4vovTn++0Onaau7jUt9bmUtAN6IK5/3AzRpNC5eYu2vMEKPI5H+MnpDHh6NgZ2YMkN/9kbggEgEYLgnl9TeTR3v772A0QJYsP5Wt4IqY2nVvOCHEj6aVOaASAOV1HAJuVDsuirLrBgmdqh0eXfR3ePd4f6q0XN8S/jqKIBATGqAQp2MS4xLjAuMC0wsgECAAC6AQIAAA==",
+    "headerType": "Header",
+    "headerHash": "X4vFKWPBC7txMzSSKDkaEoLZ75S+39py5YrPVpQhu9Y=",
+    "body": {}
+  },
+  "transactionPool": {},
```

Comment:
i thinks it's good to have here an example also with `transactionPool` data

Reply:
added `transactionPool`